### PR TITLE
Added secrects doc

### PR DESCRIPTION
**Pull request by jdesouza**
This PR fixes #

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### CODE REVIEW

File: docs/configure/automation/integrations.md [hunk 1]
```diff
@@ -108,3 +108,31 @@ sendHTTPRequest("POST", "https://example.com/action-item", {
   body: JSON.stringify(ActionItem),
 });
 ```
+
+## Adding and using secrets
```
Comment (mhoss019) [inline docs/configure/automation/integrations.md:112]:
```suggestion
## Adding and Using Secrets
```

File: docs/configure/automation/integrations.md [hunk 2]
```diff
@@ -108,3 +108,31 @@ sendHTTPRequest("POST", "https://example.com/action-item", {
   body: JSON.stringify(ActionItem),
 });
 ```
+
+## Adding and Using Secrets
+Users can save secrets stored encrypted into Insights and use them, when creating automation rules, using `getSecret` function. For example:
```
Comment (mhoss019) [inline docs/configure/automation/integrations.md:113]:
```suggestion
Users can store encrypted secrets into Insights and use them when creating Automation Rules using the `getSecret` function. 
```

File: docs/configure/automation/integrations.md [hunk 3]
```diff
@@ -108,3 +108,31 @@ sendHTTPRequest("POST", "https://example.com/action-item", {
...
+
+Create secret endpoint:
+```js
+curl 'https://insights.fairwinds.com/v0/organizations/acme-co/secrets/bulk' \
```
Comment (mhoss019) [inline docs/configure/automation/integrations.md:117]:
```suggestion
curl 'https://insights.fairwinds.com/v0/organizations/$YOUR_ORG/secrets/bulk' \
```

File: docs/configure/automation/integrations.md [hunk 4]
```diff
@@ -108,3 +108,31 @@ sendHTTPRequest("POST", "https://example.com/action-item", {
...
+  -H 'accept: application/json, text/plain, */*' \
+  -H 'content-type: application/json' \
+  ...
+    --data-raw '[{"key": "MY_SECRET", "value": "pwd-12345"}]'
```
Comment (mhoss019) [inline docs/configure/automation/integrations.md:121]:
```suggestion
    --data-raw '[{"key": "$KEY", "value": "$VALUE"}]'
```

File: docs/configure/automation/integrations.md [hunk 5]
```diff
@@ -108,3 +108,31 @@ sendHTTPRequest("POST", "https://example.com/action-item", {
   body: JSON.stringify(ActionItem),
 });
 ```
+
+## Adding and Using Secrets
+Users can save secrets stored encrypted into Insights and use them, when creating automation rules, using `getSecret` function. For example:
+
+Create secret endpoint:
```
Comment (mhoss019) [inline docs/configure/automation/integrations.md:115]:
```suggestion
Create a secret:
```

File: docs/configure/automation/integrations.md [hunk 6]
```diff
@@ -108,3 +108,31 @@ sendHTTPRequest("POST", "https://example.com/action-item", {
...
+
+## Adding and Using Secrets
+Users can store encrypted secrets into Insights and use them when creating Automation Rules using the `getSecret` function. 
+
+Create a secret:
+```js
+curl 'https://insights.fairwinds.com/v0/organizations/$YOUR_ORG/secrets/bulk' \
+  -H 'accept: application/json, text/plain, */*' \
+  -H 'content-type: application/json' \
+  ...
```
Comment (mhoss019) [inline docs/configure/automation/integrations.md:120]: What is this ... ?

File: docs/configure/automation/integrations.md [hunk 7]
```diff
@@ -108,3 +108,31 @@ sendHTTPRequest("POST", "https://example.com/action-item", {
...
+    --data-raw '[{"key": "$KEY", "value": "$VALUE"}]'
+```
+
+Listing secrets keys endpoint:
+```js
+curl 'https://insights.fairwinds.com/v0/organizations/acme-co/secrets/bulk'
+...
```
Comment (mhoss019) [inline docs/configure/automation/integrations.md:127]: Not sure what the ... is? I think it should be removed

File: docs/configure/automation/integrations.md (hunk 6 above)
Comment (jdesouza) [reply docs/configure/automation/integrations.md:120]: cookis and other info

File: docs/configure/automation/integrations.md [hunk 8]
```diff
@@ -108,3 +108,31 @@ sendHTTPRequest("POST", "https://example.com/action-item", {
...
+curl 'https://insights.fairwinds.com/v0/organizations/$YOUR_ORG/secrets/bulk' \
+  -H 'accept: application/json, text/plain, */*' \
+  -H 'content-type: application/json' \
+  ...
+    --data-raw '[{"key": "$KEY", "value": "$VALUE"}]'
+```
+
+Listing secrets keys endpoint:
+```js
+curl 'https://insights.fairwinds.com/v0/organizations/acme-co/secrets/bulk'
```
Comment (mhoss019) [inline docs/configure/automation/integrations.md:126]:
```suggestion
curl 'https://insights.fairwinds.com/v0/organizations/$YOUR_ORG/secrets/bulk'
```

File: docs/configure/automation/integrations.md (hunk 7 above)
Comment (jdesouza) [reply docs/configure/automation/integrations.md:127]: Cookies and other http parameters

File: docs/configure/automation/integrations.md [hunk 9]
```diff
@@ -108,3 +108,31 @@ sendHTTPRequest("POST", "https://example.com/action-item", {
...
+Create a secret:
+```js
+curl 'https://insights.fairwinds.com/v0/organizations/$YOUR_ORG/secrets/bulk' \
+  -H 'accept: application/json, text/plain, */*' \
+  -H 'content-type: application/json' \
+  ...
+    --data-raw '[{"key": "$KEY", "value": "$VALUE"}]'
+```
+
+Listing secrets keys endpoint:
```
Comment (mhoss019) [inline docs/configure/automation/integrations.md:124]:
```suggestion
To list current secrets:
```

File: docs/configure/automation/integrations.md (hunk 6 above)
Comment (mhoss019) [reply docs/configure/automation/integrations.md:120]: I would remove it then

File: docs/configure/automation/integrations.md (hunk 7 above)
Comment (mhoss019) [reply docs/configure/automation/integrations.md:127]: I would remove it then

File: docs/configure/automation/integrations.md [hunk 10]
```diff
@@ -108,3 +108,31 @@ sendHTTPRequest("POST", "https://example.com/action-item", {
...
+## Adding and Using Secrets
+Users can store encrypted secrets into Insights and use them when creating Automation Rules using the `getSecret` function. 
+
+Create a secret:
+```js
+curl 'https://insights.fairwinds.com/v0/organizations/$YOUR_ORG/secrets/bulk' \
+  -H 'accept: application/json, text/plain, */*' \
+  -H 'content-type: application/json' \
+  ...
+    --data-raw '[{"key": "$KEY", "value": "$VALUE"}]'
```
Comment (mhoss019) [inline docs/configure/automation/integrations.md:121]: I think the indentation is off here by one extra space.

**Review by mhoss019 — APPROVED** [review]

**Pull request merged**